src/core/shaderlib/picking/picking.js

```diff
@@ -1,105 +0,0 @@
-import log from '../../utils/log';
```

Comment:
Assume you carefully compared this and luma.gl so the resulting code in luma contains the best of both?

Reply:
Yep, `deck.gl/src/core/shaderlib/picking/picking.js` and  `luma.gl/src/shadertools/modules/picking/picking.js` are identical except 'DEFAULT_HIGHLIGHT_COLOR', in deck.gl it was `[0, 64, 128, 64]` and in luma.gl is `[0, 255, 255, 255]`, and I think luma's version is correct as 64 alpha blends the content behind highlighted object and might be confusing.